### PR TITLE
switch ci from travis to GitHub Actions

### PR DESCRIPTION
**Pull request by namusyaka**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        # Due to https://github.com/actions/runner/issues/849, we have to use quotes for '3.0'
+        ruby: [2.6, 2.7, '3.0', head, jruby-head]
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+        bundler-cache: true
+    - name: Install dependencies
+      run: bundle install
```
Comment (dentarg) [inline .github/workflows/test.yml:31]: We can omit `bundle install` when we use `bundler-cache: true` (https://github.com/ruby/setup-ruby#single-job)

Comment (namusyaka) [reply .github/workflows/test.yml:31]: ack, thanks.

**Comment by namusyaka** [comment]
We also have to work on slack notification. I'm not sure what plugin should be used.

**Comment by dentarg** [comment]
> We also have to work on slack notification

What about https://github.com/integrations/slack#github--slack-integration (https://slack.github.com/), can that be used?

What's the Slack where it is going to be used? Is there one for the Sinatra org?

**Comment by dentarg** [comment]
> What's the Slack where it is going to be used? Is there one for the Sinatra org?

I'll answer myself: https://github.com/sinatra/sinatra#further-reading has links to it ("Sinatra & Friends", https://sinatrarb.slack.com/)

**Comment by namusyaka** [comment]
>What about https://github.com/integrations/slack#github--slack-integration (https://slack.github.com/), can that be used?

Looks promising. Thanks.

>What's the Slack where it is going to be used? Is there one for the Sinatra org?
I'll answer myself: https://github.com/sinatra/sinatra#further-reading has links to it ("Sinatra & Friends", https://sinatrarb.slack.com/)

True. But I noticed that mustermann does not integrate with the Slack properly. I'll skip this once.

**Pull request merged**